linux_os/guide/system/auditing/configure_auditd_data_retention/var_audispd_remote_server.var

```diff
@@ -14,4 +14,4 @@ type: string
 interactive: true
 
 options:
-    default: myhost.mydomain.com
+    default: logcollector
```

Comment:
The test scenarios in `linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_configure_remote_server/tests` need to be updated as well.

Reply:
I'll fix shortly.